dev/integration_tests/ios_add2app/Podfile

```diff
@@ -1,26 +1,16 @@
 platform :ios, '12.0'
 
-# Prevent Cocoapods from embedding a second Flutter framework and causing an error with the new Xcode build system.
-install! 'cocoapods', :disable_input_output_paths => true
-
 flutter_application_path = 'flutterapp/'
-framework_dir = File.join(flutter_application_path, '.ios', 'Flutter')
 
-engine_dir = File.join(framework_dir, 'engine')
-if !File.exist?(engine_dir)
-  # Copy the debug engine to have something to link against if the xcode backend script has not run yet.
-  debug_framework_dir = File.join(flutter_root(flutter_application_path), 'bin', 'cache', 'artifacts', 'engine', 'ios')
-  FileUtils.mkdir_p(engine_dir)
-  FileUtils.cp_r(File.join(debug_framework_dir, 'Flutter.framework'), engine_dir)
-  FileUtils.cp(File.join(debug_framework_dir, 'Flutter.podspec'), engine_dir)
-end
+load File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')
 
 target 'ios_add2app' do
-  eval(File.read(File.join(flutter_application_path, '.ios', 'Flutter', 'podhelper.rb')), binding)
+  install_flutter_plugin_pods(flutter_application_path)
+  install_flutter_application_pod(flutter_application_path, 'ios_add2app_flutter')
```

Comment:
This could have called `install_all_flutter_pods flutter_application_path` if the application path matched the project name.  I expect most of them to match, but I added this workaround in case they don't.

Reply:
Right now `install_flutter_plugin_pods` is used in the sub-project Podfile for the stand-alone Runner app to just pull in Flutter and the plugins, but not the App.framework.
BUT this question just made me reconsider this.  Why is the Runner sub-app special?  Why not let CocoaPods hande embedding Flutter and App there too?  Why the generated xcconfig?  Why add Flutter and App to the app itself?

The sub-app works if I let the App be embedded as a pod, so why not do that?  When I do I can remove:
1. All the script phases that call xcode_build.  CocoaPods handles thinning.
2. The embed build phases.
3. Flutter.framework and App.framework as files in the Xcode project

And if that's the case, is there any reason to not do that for all Flutter Runner apps, not just module versions?
@dnfield

Reply:
Hm, although right now we aren't running pod install if there are no plugins, and I wouldn't want to add more CocoaPods paths...

Reply:
Though solving this (not embedding the frameworks explicitly if we're in CocoaPod world, and letting CocoaPod embed and thin)  would fix the underlying issue of https://github.com/flutter/flutter/issues/20685, which is that two build phases want to output a Flutter.framework.

Reply:
> BUT this question just made me reconsider this. Why is the Runner sub-app special? Why not let CocoaPods hande embedding Flutter and App there too? Why the generated xcconfig? Why add Flutter and App to the app itself?

I think it might be for the minor shortcut where the Runner can run for people without plugins and without installing cocoapods